internal/server/iito.go

```diff
@@ -1,10 +1,18 @@
 package server
 
+import (
```

Comment:
I feel we want to ignore dead clients?

Reply:
I tried to add it everywhere I could think of but I can see through once again?

Reply:
yeha tbh, done is better than perfect for now..